xline/src/storage/kvwatcher.rs

```diff
@@ -424,4 +597,50 @@ mod test {
         }
         handle.abort();
     }
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 10)]
+    async fn test_victim() {
```

Comment:
Is there a test to cover watch cancel logic?

Reply:
Added